src/main/java/com/dao/homework/dao/jdbc/DriverDaoJdbcImpl.java

```diff
@@ -0,0 +1,123 @@
+package com.dao.homework.dao.jdbc;
+
+import com.dao.homework.dao.DriverDao;
+import com.dao.homework.exceptions.DataProcessingException;
+import com.dao.homework.lib.Dao;
+import com.dao.homework.model.Driver;
+import com.dao.homework.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class DriverDaoJdbcImpl implements DriverDao {
+    private static final String TABLE_NAME = "drivers";
+    private static final String DRIVERS_NAME = "drivers_name";
+    private static final String LICENSE_NUMBER = "license_number";
+    private static final String DRIVERS_ID = "drivers_id";
+    private static final String DELETED = "deleted";
```

Comment:
if I have written a comment and you agree with it then let's solve this in each similar places(not only the exact place where the comment has been written)